src/diffusers/loaders.py

```diff
@@ -1657,6 +1656,12 @@ def _convert_kohya_lora_to_diffusers(cls, state_dict):
                     te2_state_dict[diffusers_name] = state_dict.pop(key)
                     te2_state_dict[diffusers_name.replace(".down.", ".up.")] = state_dict.pop(lora_name_up)
 
+            # let's rename alphas here
+            if lora_name_alpha in state_dict:
+                alpha = state_dict.pop(lora_name_alpha).item()
+                new_name = "unet." + diffusers_name.split(".lora.")[0] + ".alpha"
```

Comment:
This only concerns UNet, what about the text encoder?

Reply:
Good point maybe something like:
```suggestion
                prefix = "" if diffusers_name.startswith("text_encoder") else "unet."
                new_name = prefix + diffusers_name.split(".lora.")[0] + ".alpha"
```
?

Reply:
See if the current works?